tests/util/test_condition_tools.py

```diff
@@ -188,19 +191,7 @@ def test_agg_sig_unsafe_restriction(self, opcode: ConditionOpcode) -> None:
 
 
 class TestPkmPairsForConditionDict:
-    @pytest.mark.parametrize(
-        "opcode",
-        [
-            ConditionOpcode.AGG_SIG_PARENT,
-            ConditionOpcode.AGG_SIG_PUZZLE,
-            ConditionOpcode.AGG_SIG_AMOUNT,
-            ConditionOpcode.AGG_SIG_PUZZLE_AMOUNT,
-            ConditionOpcode.AGG_SIG_PARENT_AMOUNT,
-            ConditionOpcode.AGG_SIG_PARENT_PUZZLE,
-            ConditionOpcode.AGG_SIG_ME,
-        ],
-    )
```

Comment:
don't you need to parameterize on all conditions still?

Reply:
There is a similarly named function with the parameters, this one just covers the invalid condition case for `pkm_pairs_for_conditions_dict()` and the only use for the opcodes was in constructing the additional data dictionary.